components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -325,11 +324,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
   cell.backgroundColor = self.backgroundColor;
   cell.actionFont = self.actionFont;
   cell.accessibilityIdentifier = action.accessibilityIdentifier;
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  cell.inkColor = self.inkColor;
-  cell.enableRippleBehavior = self.enableRippleBehavior;
-#pragma clang diagnostic pop
+  cell.enableRippleBehavior = YES;
```

Comment:
Can this property be removed on the cell? Fine as a fast-follow PR.

Reply:
Yep, added a comment in this PR's description to do so.